Musescore.config

```diff
@@ -0,0 +1,2 @@
+// Add predefined macros for your project here. For example:
+// #define THE_ANSWER 42
```

Comment:
Is this commit correct: 57d3883 , as below you give 77d3883 ? - I didn't find either of them in the master commits.